chromium_src/components/search_engines/template_url_prepopulate_data.cc

```diff
@@ -27,6 +30,36 @@ namespace TemplateURLPrepopulateData {
 
 namespace {
 
+#define CHAR_A 'A'
+#define CHAR_B 'B'
+#define CHAR_C 'C'
+#define CHAR_D 'D'
+#define CHAR_E 'E'
+#define CHAR_F 'F'
+#define CHAR_G 'G'
+#define CHAR_H 'H'
+#define CHAR_I 'I'
+#define CHAR_J 'J'
+#define CHAR_K 'K'
+#define CHAR_L 'L'
+#define CHAR_M 'M'
+#define CHAR_N 'N'
+#define CHAR_O 'O'
+#define CHAR_P 'P'
+#define CHAR_Q 'Q'
+#define CHAR_R 'R'
+#define CHAR_S 'S'
+#define CHAR_T 'T'
+#define CHAR_U 'U'
+#define CHAR_V 'V'
+#define CHAR_W 'W'
+#define CHAR_X 'X'
+#define CHAR_Y 'Y'
+#define CHAR_Z 'Z'
+#define CHAR(ch) CHAR_##ch
+#define CODE_TO_ID(code1, code2)\
```

Comment:
why can't we just use `country_codes::CountryCharsToCountryID('D', 'E')` and `country_codes::CountryCharsToCountryID('F', 'R')`
https://github.com/brave/brave-core/blob/7a477860bdf83964a1a43476c4defec91675368d/chromium_src/components/search_engines/template_url_prepopulate_data.cc#L102

Reply:
I'll try

Reply:
updated and works, thanks.